src/app/cube/core/usage-stats/usage-stats.service.ts

```diff
@@ -28,7 +29,7 @@ export class UsageStatsService {
         )
         .toPromise(),
       this.http
-        .get<{metrics: any}>(STATS_ROUTES.LIBRARY_METRICS)
```

Comment:
Just confirming these are the same routes?

Reply:
Yes.
STATS_ROUTES.LIBRARY_METRICS = ${environment.apiURL}/learning-objects/metrics
METRIC_ROUTES.GET_LEARNING_OBJECT_METRICS() =  ${environment.apiURL}/learning-objects/metrics